assets/js/tutorials/freeze-individual-line.js

```diff
@@ -77,7 +77,7 @@ $(document).ready(() => {
         "peer": peer.address,
         "ledger_index": "validated"
       })
-    } catch(e) {
```

Comment:
Noticed & fixed this typo while looking at the other code for reference. Otherwise unrelated.